.github/workflows/main.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   build:
-    uses: mrbeam/mrb-workflows/.github/workflows/build.yml@main
+    uses: mrbeam/mrb-workflows/.github/workflows/build.yml@python2.7
```

Comment:
shouldn't this be merged back to "main" once approved?

Reply:
No, this will only take affect for the MrBeamPlugin as this is the only one with python2.7 were we still have some developments.